packages/form-config/src/components/Antd/UploadForm/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -0,0 +1,333 @@
+          "offset": 0,
+          "span": 8,
+        },
+        "sm": Object {
+          "offset": 2,
+          "span": 10,
+        },
+        "xs": Object {
+          "offset": 0,
+          "span": 16,
+        },
+      }
+    }
+    onFinish={[Function]}
+    wrapperCol={
+      Object {
+        "lg": Object {
+          "span": 10,
+        },
+        "md": Object {
+          "span": 12,
+        },
+        "sm": Object {
+          "span": 14,
+        },
+        "xs": Object {
+          "span": 24,
+        },
+      }
+    }
+  >
+    <FormItem
+      label="Form Name"
+      name="form_name"
+      rules={
+        Array [
+          Object {
+            "message": "Form Name is required",
+            "required": true,
+          },
+        ]
+      }
+    >
+      <Input
+        disabled={true}
+        type="text"
+      />
+    </FormItem>
+    <FormItem
+      label="Module"
+      name="module"
+    >
+      <Input
+        disabled={true}
+        type="text"
+      />
+    </FormItem>
+    <FormItem
+      label="Related to"
+      name="form_relation"
+    >
+      <Input
+        disabled={true}
+        type="text"
+      />
+    </FormItem>
+    <FormItem
+      getValueFromEvent={[Function]}
+      label="Form"
+      name="form"
+      rules={
+        Array [
+          Object {
+            "message": "Form is required",
+            "required": true,
+          },
+        ]
+      }
+      valuePropName="fileList"
+    >
+      <Upload
+        accept=""
+        action=""
+        beforeUpload={[Function]}
+        className=""
+        data={Object {}}
+        disabled={false}
+        listType="text"
+        multiple={false}
+        onRemove={[Function]}
+        showUploadList={true}
+        supportServerRender={true}
+        type="select"
+      >
+        <Button
+          block={false}
+          ghost={false}
+          htmlType="button"
+          icon={<ForwardRef(UploadOutlined) />}
+          loading={false}
+        >
+          Click to Upload
+        </Button>
+      </Upload>
+    </FormItem>
+    <FormItem
+      wrapperCol={
+        Object {
+          "lg": Object {
+            "offset": 6,
+            "span": 14,
+          },
+          "md": Object {
+            "offset": 8,
+            "span": 16,
+          },
+          "sm": Object {
+            "offset": 12,
+            "span": 24,
+          },
+          "xs": Object {
+            "offset": 0,
+            "span": 16,
+          },
+        }
+      }
+    >
+      <Button
+        block={false}
+        ghost={false}
+        htmlType="submit"
+        loading={false}
+        type="primary"
+      >
+        Upload form
+      </Button>
+    </FormItem>
+  </ForwardRef(InternalForm)>,
+}
+`;
+
+exports[`components/UploadFile edits form correctly: title 1`] = `
+Object {
+  "children": "Upload Form",
+  "level": 3,
+}
+`;
+
+exports[`components/UploadFile renders correctly: card 1`] = `
```

Comment:
one of those relatively bigger snapshots.

Reply:
@p-netm I don't think this snapshot can be reduced further. If you look closely, it's only a snapshot of the props, I feel if I reduce it, i might have to end up missing to test some props

Reply:
@p-netm Do you have a way we can make it shorter without losing to test some stuff

Reply:
@p-netm I have split the other snapshots but I can't reduce this one